simplepbr/__init__.py

```diff
@@ -40,7 +40,12 @@ def _add_shader_defines(shaderstr, defines):
 
 
 def _load_shader_str(shaderpath, defines=None):
-    shader_dir = os.path.join(os.path.dirname(__file__), 'shaders')
+    # per rdb -- current directory is platform-specific resources
```

Comment:
I don't believe we need this comment

Reply:
clean version of this is in https://github.com/Moguri/panda3d-simplepbr/pull/32